de-lib/src/main/java/org/iplantc/de/tools/client/presenter/ManageToolsViewPresenter.java

```diff
@@ -308,16 +310,28 @@ public HandlerRegistration addSelectionChangedHandler(SelectionChangedEvent.Sele
 
     @Override
     public void onShowToolInfo(ShowToolInfoEvent event) {
-       toolInfoDialogProvider.get(new AsyncCallback<ToolInfoDialog>() {
-           @Override
-           public void onFailure(Throwable throwable) {
+        dcService.getAppsForTool(event.getTool().getId(), new AppsCallback<List<App>>() {
+
+            @Override
+            public void onFailure(Integer statusCode, Throwable exception) {
```

Comment:
Error from the service. This is the `onFailure` for `dcService.getAppsForTool`, so it could be anything, like a 404 if the service can't find this tool-id for some reason, or if the response doesn't pass the format validation, etc.